AUTHORS

```diff
@@ -34,3 +34,4 @@ Kevin Watts <kevinwwatts@gmail.com>
 Jie Yu <yujie.jay@gmail.com>
 Andrew Z Allen <me@andrewzallen.com>
 Ming Zhao <mzhao@luminatewireless.com>
+Jacob Parker <jacob@solidangle.ca>
```

Comment:
Just to be clear, the AUTHORS file contain the copyright holders.

Reply:
Yup; I assumed the procedure for an individual contributor (not someone contributing on behalf of their employer, or with a particular employment agreement about how this works etc.) was to add to both files. If I got that wrong let me know.

Reply:
I'm not sure, but I don't think you need to add yourself here. Did some documentation advise it?

Reply:
Nope, I think I just misunderstood. I'll remove this change.